Automattic/wp-calypso
# Create ProductCard UI component based on the PlansSingleProduct

#### Changes proposed in this Pull Request

* Migrate code from `ProductCard`
* Add readme file
* Add `example.jsx` to be used in devdocs
* Migrate `ProductCardPriceGroup`
* Clean up props list

**Please note that there are still some parts of the product card missing. They will be added in the next PR.** 

**Mobile:**
![Screenshot 2019-10-15 at 17 20 07](https://user-images.githubusercontent.com/478735/66845423-4684e180-ef70-11e9-8649-3451c763a094.png)
![Screenshot 2019-10-15 at 17 20 44](https://user-images.githubusercontent.com/478735/66845429-48e73b80-ef70-11e9-8d60-374e97738473.png)

**Desktop:**
![Screenshot 2019-10-15 at 17 21 04](https://user-images.githubusercontent.com/478735/66845443-4dabef80-ef70-11e9-889f-692ea96a1ba8.png)

#### Testing instructions

* Checkout this branch on your local dev environment.
* Run `npm start`.
* Go to the new component's devdoc: http://calypso.localhost:3000/devdocs/design/single-product-plan
* Confirm that the product card looks and works well on various screen sizes

**Master thread:** p1HpG7-7nT-p2

Fixes no known issue.
